core/src/main/java/com/linecorp/armeria/client/HttpResponseWrapper.java

```diff
@@ -62,15 +64,22 @@ class HttpResponseWrapper implements StreamWriter<HttpObject> {
     private boolean done;
     private boolean closed;
 
-    HttpResponseWrapper(DecodedHttpResponse delegate, EventLoop eventLoop, ClientRequestContext ctx,
+    HttpResponseWrapper(AbstractHttpRequestHandler requestHandler,
+                        DecodedHttpResponse delegate, EventLoop eventLoop, ClientRequestContext ctx,
                         long responseTimeoutMillis, long maxContentLength) {
+        this.requestHandler = requestHandler;
         this.delegate = delegate;
         this.eventLoop = eventLoop;
         this.ctx = ctx;
         this.maxContentLength = maxContentLength;
         this.responseTimeoutMillis = responseTimeoutMillis;
     }
 
+    AbstractHttpRequestHandler requestHandler() {
+        assert requestHandler != null;
+        return requestHandler;
+    }
```

Comment:
What do you think about replacing this method as `handle100Continue()`, given that the caller of this method always calls `requestHandler.handle100Continue()`?

Reply:
It would be nice if we could call it with `res.handle100Continue()`, which would reduce the cognitive load on the reader.
Let me modify it.